Valet/VALValet.m

```diff
@@ -237,37 +237,35 @@ + (nullable NSString *)_sharedAccessGroupPrefix;
 
 #pragma mark - Initialization
 
-- (nullable instancetype)initWithIdentifier:(nonnull NSString *)identifier accessibility:(VALAccessibility)accessibility;
+- (nonnull instancetype)initWithIdentifier:(nonnull NSString *)identifier accessibility:(VALAccessibility)accessibility;
```

Comment:
this method can return `nil`, so this needs to remain `nullable`. `VALCheckCondition` returns `nil` if the condition is not met.